api/app/utils/emailer.py

```diff
@@ -8,6 +8,54 @@
 from email.mime.base import MIMEBase
 from email import encoders
 from flask import g, request
+from app.email_template.repository import EmailRepository as email_repository
+from app.users.repository import UserRepository as user_repository
+from app.events.repository import EventRepository as event_repository
+
+def email_user(
+    email_template_key, 
+    template_parameters=None, 
+    event_id=None,
+    event=None,
+    user_id=None, 
+    user=None, 
```

Comment:
I'm wondering if it would be cleaner to just pass in the ids or just the objects. I'm leaning more towards the objects since usually I think we do have the object retrieved already or was there a reason you did it this way?

Reply:
I thought it would be easier for the caller, but yes it does seem that the caller always has the objects anyway so updated as per your suggestion